### PR TITLE
tests/testpython.py: Covert to Python 3

### PR DESCRIPTION
**Pull request by travier**
See: https://bugzilla.redhat.com/show_bug.cgi?id=2249819
See: https://fedoraproject.org/wiki/Changes/RetirePython2.7

### CODE REVIEW

**Comment by travier** [comment]
Untested.

File: tests/testpython.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python2
+#!/usr/bin/python
 
 import importme
 print importme.foo
```
Comment (hroncok) [inline tests/testpython.py:4]: This is Python 2 syntax.

**Comment by hroncok** [comment]
I've read https://github.com/flatpak/flatpak-builder/commit/9c57a5751ee5c0403700c9da0d35ba56b08c1995 and porting this file to Python 3 makes very little sense to me.

**Comment by hroncok** [comment]
To clarify: The purpose of the test seems to be a script with python2 shebang.

If you change the shebang (and port the script to python 3), the purpose of the test is defeated.

The test is skipped when python2 is not available. To fix https://bugzilla.redhat.com/show_bug.cgi?id=2249819 IMHO the Fedora package should filter the automatic dependency out instead.

**Comment by debarshiray** [comment]
> To clarify: The purpose of the test seems to be a script with python2 shebang.
> 
> If you change the shebang (and port the script to python 3), the purpose of the test is defeated.

Thanks for digging into the code.  I never got around to digging into what commit 9c57a5751ee5c0403700c9da0d35ba56b08c1995 and commit 006d9a19270416f7c57b7ba4220ef8ba306e8b94 were actually trying to accomplish, which was made worse by my almost non-existent knowledge of Python these days.  So, thanks for doing that.

> The test is skipped when python2 is not available. To fix https://bugzilla.redhat.com/show_bug.cgi?id=2249819 IMHO the Fedora package should filter the automatic dependency out instead.

Okay!

**Review by debarshiray — COMMENTED** [review]
If this commit is *really* needed, then there's a minor typo in the commit message.  It should be *convert*, not *covert*.  :)

**Comment by TingPing** [comment]
I'll close this since it is a python2 specific test. Someday it can just be removed entirely but it costs nothing to leave it there, and `python2` can and will still be used in flatpak for a least a bit.

**Comment by travier** [comment]
👍🏻 Thanks for digging!

**Pull request closed without merging**